apps/eslint/index.js

```diff
@@ -0,0 +1,63 @@
+module.exports = {
+  rules: {
+    'style-blocks-below-class': {
+      meta: {
+        fixable: 'code'
+      },
+      create: function(context) {
+        return {
+          Program(node) {
+            const styleBlock = node.body.filter(
+              item =>
+                item.type === 'VariableDeclaration' &&
+                item.declarations[0].id.name === 'styles'
+            )[0];
+
+            const classDeclarations = node.body.filter(
+              item =>
+                item.type === 'ClassDeclaration' ||
+                (item.type === 'ExportDefaultDeclaration' &&
+                  item.declaration.type === 'ClassDeclaration')
+            );
+
+            // if the program contains a style block and class declaration
+            if (styleBlock && classDeclarations.length) {
```

Comment:
nit: we could reduce indentation here if we invert this conditional and `return null;` early